README.md

```diff
@@ -3,50 +3,63 @@
 > This cheat sheet summarize important steps you should follow to promote your open-source project in the best conditions.
 
 List of availables languages:
+
 - [English](./README.md)
 - [Français](./README-fr.md)
 
+---
+
+# Cheat sheet
```

Comment:
Is this title relevant as we already mention in the description that it's a cheat sheet ?

Reply:
You're right, I didn't see that. However I think we should put a title for this section because I add Useful links section. Any idea ? Maybe "Steps" ?

Reply:
Not relevent since I'll delete Useful Links section.